### PR TITLE
[Product description AI] Build celebration screen

### PR DESCRIPTION
**Pull request by selanthiraiyan**
Part of: #9934

## Description

Adds the code for the product description generation celebration screen. 

## Testing instructions
- You can check the UI by using SwiftUI preview `ProductDescriptionGenerationCelebrationView`. 
- No further testing needed as we are not using this view as of now. 

## Screenshots
Screenshot 2023-06-13 at 12 35 39 PM

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Resources/Images.xcassets/ai-description-celebration.imageset/ai-description-celebration.pdf
Comment (selanthiraiyan) [inline WooCommerce/Resources/Images.xcassets/ai-description-celebration.imageset/ai-description-celebration.pdf:1]: I have asked for the dark mode equivalent to the design team. I will add it in a future PR if available.

File: WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Celebration view presented when AI generation is used for the first time
+struct ProductDescriptionGenerationCelebrationView: View {
+    private var viewModel: ProductDescriptionGenerationCelebrationViewModel
```
Comment (itsmeichigo) [inline WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift]: Nit: if this view model is not an `@ObservedObject`, we can keep it a constant instead.

File: WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            Image(uiImage: viewModel.celebrationImage)
+
+            Text(viewModel.greatStartLabel)
+                .headlineStyle()
+                .multilineTextAlignment(.center)
+
+            Text(viewModel.instructionsLabel)
+                .foregroundColor(Color(.text))
+                .subheadlineStyle()
+                .multilineTextAlignment(.center)
```
Comment (itsmeichigo) [inline WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift]: Super nit: I notice on the design that the padding for the text is 24. We probably want to follow it to make the text stay more condensed in the center.

**Review by itsmeichigo — APPROVED** [review]
LGTM 🚢

File: WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift (hunk 1 above)
Comment (selanthiraiyan) [reply WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift]: Done in https://github.com/woocommerce/woocommerce-ios/pull/9939/commits/7802a68f902d3d596430f8cc0dba6e351524518f

File: WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift (hunk 2 above)
Comment (selanthiraiyan) [reply WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift]: Good catch. Done in https://github.com/woocommerce/woocommerce-ios/pull/9939/commits/3b8310f5a37585f199a78ee904a985f21ffc0739

**Pull request merged**